examples/docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/schedules/schedules.py

```diff
@@ -48,7 +48,7 @@ def configurable_job():
 
 @schedule(job=configurable_job, cron_schedule="0 0 * * *")
 def configurable_job_schedule(context: ScheduleEvaluationContext):
-    scheduled_date = context.scheduled_execution_time.strftime("%Y-%m-%d")
+    scheduled_date = context.scheduled_execution_time.strftime("%Y-%m-%d")  # type: ignore (didactic)
```

Comment:
why is this a type error?

Reply:
Because `context.scheduled_execution_time` can be `None`.